generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -28,8 +28,10 @@ import { <%=angularXAppName%>AdminModule } from './admin/admin.module';
 <%_ if (authenticationType !== 'oauth2') { _%>
 import { <%=angularXAppName%>AccountModule } from './account/account.module';
 <%_ } _%>
+<%_ if (authenticationType !== 'jwt') { _%>
```

Comment:
AuthServerProvider is listed in [`shared/shared.module.ts`](https://github.com/clarenced/generator-jhipster/blob/e8324351d0fc6669338b4ff5d7c6c1323e9b35fa/generators/client/templates/angular/src/main/webapp/app/shared/_shared.module.ts#L26-L28) instead of `app.module.ts`.  This way you also don't have to check the authentication type either. So changes needed:

 - Remove changes to `app.module.ts`
 - Remove `<%_ if (!skipServer) { _%>` around `AuthServerProvider` in `shared/shared.module.ts`

Reply:
@ruddell OK I will do it